spec/helpers/helper_methods.rb

```diff
@@ -0,0 +1,15 @@
+module HelperMethods
+  def login_admin
+    current_controller = @controller
+    @controller = AdminSessionsController.new
+
+    admin = create(:admin)
```

Comment:
Should we pass `admin` as an arg here instead of creating it inside the helper method? This way, we can be more explicit about creating/stubbing user objects and prevent us from mistakingly creating admins twice

Reply:
You're right, but the problem lies within the factory. I just wrote it quickly with some random data, but we can't have a factory creating the same admin over and over. I'll put in the `Faker` gem to generate random unique emails.

Reply:
You don't necessarily have to - you can use a sequence instead. Just do something like
```ruby
sequence(:email) { |n| "user_#{n}@example.com" }
```
which should generate a unique email address on its own.

Reply:
>  can't have a factory creating the same admin over and over

Why is this a problem?

Reply:
Yeah, in the end I did both - added admin as an optional argument and getting the Faker gem so we can have multiple admins when needed and getting different email addresses.